docs/installation/presto-admin-installation.rst

```diff
@@ -29,7 +29,7 @@ work.
 
 To install ``presto-admin``:
 
-1. Copy the installer ``prestoadmin-<version>-offline.tar.bz2`` to the
+1. Copy the installer ``prestoadmin-<version>-offline.tar.gz`` to the
 location where you want ``presto-admin`` to run.
 Note that ``presto-admin`` does not have to be on the same node(s)
```

Comment:
Change this line in second point
tar xvf prestoadmin-<version>-offline.tar.bz2
and check that the command to extract works with gz files